config/master/aws-k8s-cni.yaml

```diff
@@ -119,25 +119,24 @@ spec:
       priorityClassName: "system-node-critical"
       serviceAccountName: aws-node
       hostNetwork: true
-      initContainers:
-      - name: aws-vpc-cni-init
-        image: "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.12.0"
-        env:
-          - name: DISABLE_TCP_EARLY_DEMUX
-            value: "false"
-          - name: ENABLE_IPv6
-            value: "false"
-        securityContext:
+      containers:
```

Comment:
Can you copy these changes to the other `config/master` files as well?
`aws-k8s-cni-cn.yaml`
`aws-k8s-cni-us-gov-east-1.yaml`
`aws-k8s-cni-us-gov-west-1.yaml`